bsp_diff/common/kernel/linux-intel-lts2021/0032-enable-vblank-for-crtc.patch

```diff
@@ -0,0 +1,291 @@
+From 3920946c01d411f379eb2fa477f8db21ee9ada70 Mon Sep 17 00:00:00 2001
+From: Hang Liu<hang1.liu@intel.com>
+Date: Sat, 1 Jul 2023 12:30:18 +0800
+Subject: [PATCH 3/3] enable vblank for crtc
+
+virtio gpu backend inject virtual vblank
```

Comment:
as the number of characters/line in commit message is <=72 characters. Please make use of the full 72characters which can reduce the number of lines in commit message.

Reply:
OK